sortinghat/core/api.py

```diff
@@ -868,7 +869,7 @@ def _find_individuals(from_uuids, to_uuid):
             if from_uuid == '':
                 raise InvalidValueError(msg="'from_uuid' cannot be an empty string")
             if from_uuid == to_uuid:
-                raise InvalidValueError(msg="'from_uuid' and 'to_uuid' cannot be equal")
+                raise EqualIndividualError(msg="'from_uuid' and 'to_uuid' cannot be equal")
```

Comment:
I'm not sure if this is the best error message because the function receives `from_uuids` which is a list. I think it should be better something like `to_uuid cannot be part of `from_uuids`. I also think it would be useful to have the hash `uuid` in the message.